routes/channel.js

```diff
@@ -43,6 +53,35 @@ function getStatus(withTree, req, res) {
 	})
 }
 
+// Implementation of get validator messages last recent N
+// validator messages
```

Comment:
shouldn't it be "getvalidatorMessages"
also you do not need to start with Promise.resolve(), you can just start with the find().. chain 
and don't forget .catch(next) as all the other methods do

Reply:
done